libs/langchain/langchain/chains/structured_output/base.py

```diff
@@ -103,17 +95,22 @@ class RecordDog(BaseModel):
     if len(openai_functions) == 1 and enforce_single_function_usage:
         llm_kwargs["function_call"] = {"name": openai_functions[0]["name"]}
     output_parser = output_parser or get_openai_output_parser(functions)
-    return prompt | llm.bind(**llm_kwargs) | output_parser
+    if prompt:
+        return prompt | llm.bind(**llm_kwargs) | output_parser
+    else:
+        return llm.bind(**llm_kwargs) | output_parser
 
 
 # TODO: implement mode='openai-tools'.
 def create_structured_output_runnable(
     output_schema: Union[Dict[str, Any], Type[BaseModel]],
     llm: Runnable,
-    prompt: BasePromptTemplate,
+    prompt: Optional[BasePromptTemplate] = None,
     *,
     output_parser: Optional[Union[BaseOutputParser, BaseGenerationOutputParser]] = None,
-    mode: Literal["openai-functions", "openai-json"] = "openai-functions",
+    mode: Literal[
+        "openai-tools", "openai-functions", "openai-json"
+    ] = "openai-functions",
```

Comment:
I wonder whether we should've made this a required argument

Reply:
yea, couldn't bc of backwards compat (mode arg was added after the function was created)